cmd/release-controller/audit_backend_gcs.go

```diff
@@ -126,7 +128,8 @@ func (b *GCSAuditStore) PutSignature(ctx context.Context, dgst string, signature
 		objectPath = path.Join("signatures", "openshift", "release", fmt.Sprintf("%s=%s", parts[0], parts[1]), fmt.Sprintf("signature-%d", index))
 	}
 	glog.V(4).Infof("Writing signature to gs://%s/%s", b.bucketName, objectPath)
-	obj := b.bucket.Object(objectPath).WithConditions(storage.IfGenerationMatch(0))
+	obj := b.bucket.Object(objectPath).If(storage.Conditions{GenerationMatch:0})
```

Comment:
I think this was the bit I wasn't certain about in the past, have we tested it?

Reply:
I saw that you had asked @smarterclayton about reviewing this particular logic in #111.  Because of that, I simply made a 1-1 change to keep the logic the same between old and new.

As far as testing goes, Im not sure we're able to get far enough in our test environment (yet) to reach this particular logic.